quinn-udp/src/unix.rs

```diff
@@ -122,6 +187,20 @@ impl UdpSocketState {
             }
         }
 
+        // Enable IP_RECVERR and IPV6_RECVERR for ICMP Errors
+        #[cfg(target_os = "linux")]
+        if is_ipv4 {
+            if let Err(_err) =
+                set_socket_option(&*io, libc::IPPROTO_IP, libc::IP_RECVERR, OPTION_ON)
+            {
+                crate::log::debug!("Failed to enable IP_RECVERR: {}", _err);
+            }
+        } else if let Err(_err) =
+            set_socket_option(&*io, libc::IPPROTO_IPV6, libc::IPV6_RECVERR, OPTION_ON)
+        {
+            crate::log::debug!("Failed to enable IPV6_RECVERR: {}", _err);
```

Comment:
`_err` can be renamed to `err`, as you _are_ using it (in the call to the `debug!` macro)